docs/core/tools/dotnet-format.md

```diff
@@ -154,3 +154,28 @@ The `dotnet format analyzers` subcommand only runs formatting rules associated w
   ```dotnetcli
   dotnet format --include ./src/ ./tests/ --exclude ./src/submodule-a/
   ```
+* Fix a specific **code style** issue:  
+
+  ```dotnetcli
+  dotnet format style --diagnostics IDExxxx --severity info
+  ```
+
+* Fix all **code style** issues that has severity `info`, `warning` or `error`:
+
+  ```dotnetcli
+  dotnet format style --severity info
+  ```
+
+* Fix a specific (non code style) analyzer issue:
+
+  ```dotnetcli  
+  dotnet format analyzers --diagnostics CAxxxx --severity warn
```

Comment:
Since this is a section showing examples, it might be better to show an actual ID number rather than CAxxxx.

Reply:
```suggestion
  dotnet format analyzers --diagnostics CA1831 --severity warn
```